src/content/components/Router/Router.js

```diff
@@ -508,6 +508,11 @@ export class Router extends React.PureComponent {
               <Route exact={true} path="/">
```

Comment:
With easily accessible arrows and Select iteration drop down, we can remove `Next Iteration` menu item in left nav (Line 163 above)

Reply:
Good idea. But if we remove the next arrow from the current iteration view, then there won't be a way to reach Next Iteration except from the dropdown picker. So do you still want to do both? Remove the next arrow from current iteration _and_ remove `Next Iteration` menu item? Might make sense to just remove one or the other

Reply:
IMO `Next Iteration` view has usually less to no bugs and having access from select dropdown should suffice.